File: blg/blg/Models/Article.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace blg.Models
{
    public class Article
    {
        public Guid GUID;
        public int? user_id;
        public int theme_id;
        public DateTime time_of_creation;
        public string title;
        public string short_description;
        public string full_description;
        public int rating;
        public bool state;

        public Article(Guid GUID, int? user_id, int theme_id, DateTime time_of_creation, string title, string short_description, string full_description, int rating, bool state)
        {
            this.GUID = GUID;
            this.user_id = user_id;
            this.theme_id = theme_id;
            this.time_of_creation = time_of_creation;
            this.title = title;
            this.short_description = short_description;
            this.full_description = full_description;
            this.rating = rating;
            this.state = state;
        }
    }
}

File: blg/blg/Scripts/article.js
"use strict";

$(function () {

      function Article() {

        //if (!arguments.length) // No args? It's getter
          //   return console.log(this);//return null;
        var _article = arguments[0]; // get article obj

        this.GUID = generateGUID();
        this.user_id = _article["userId"];
        this.theme_id = _article["theme_id"];
        this.time_of_creation = new Date();
        this.title = _article["title"];
        this.short_description = _article["short_description"][0];
        this.full_description = _article["full_description"][0];
        this.rating = 0;
        this.state = true;
    }

      function generateGUID() {
          return 'xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx'.replace(/[xy]/g, function (c) {
              var r = Math.random() * 16 | 0, v = c == 'x' ? r : (r & 0x3 | 0x8);
              return v.toString(16);
          });
      }

    // get data from url
    function parseURLParams(url) {
        var queryStart = url.indexOf("?") + 1,
            queryEnd = url.indexOf("#") + 1 || url.length + 1,
            query = url.slice(queryStart, queryEnd - 1),
            pairs = query.replace(/\+/g, " ").split("&"),
            parms = {}, i, n, v, nv;

        if (query === url || query === "") return;

        for (i = 0; i < pairs.length; i++) {
            nv = pairs[i].split("=", 2);
            n = decodeURIComponent(nv[0]);
            v = decodeURIComponent(nv[1]);

            if (!parms.hasOwnProperty(n)) parms[n] = [];
            parms[n].push(nv.length === 2 ? v : null);
        }
        return parms;
    }


    var urlParams = parseURLParams(document.URL); // get url par
    var article = new Article(urlParams); // create instance of obj

    // Insert on web page
    $("#articleTitle").text(article.title[0]);
    $("#shortDescription").text(article.short_description);
    $("#fullDescription").text(article.full_description);

    var author = (article.user_id == ((null) || (undefined))) ? "anonymous" : article.user_id;
    $("#author").text(author);
    $("#rating").text(article.rating);
    $("#dateOfCreation").text(article.time_of_creation);
});

